playwright-e2e/functional/helpers/CaseDataHelper.ts

```diff
@@ -11,14 +11,15 @@ const PARTICIPATING_COURT_REPLACEMENT: ReplacementAction[] = [
   { action: 'insert', key: 'lancashireCourtList', value: 'FR_lancashireList_1' }
 ];
 
-export class ExpressPilotHelper {
-  static async createCaseWithExpressPilot(
+export class CaseDataHelper {
+
+  private static async createCase(
     email: string,
     password: string,
     payloadPath: string,
     caseType: string,
     event: string,
-    isExpressPilot: boolean = true
+    isExpressPilot: boolean = false
   ): Promise<string> {
     const replacement: ReplacementAction[] = isExpressPilot ? PARTICIPATING_COURT_REPLACEMENT : [];
```

Comment:
Rename PARTICIPATING_COURT_REPLACEMENT to indicate it's about express pilot stuff.